draft-fossati-tls-attestation.md

```diff
@@ -828,23 +842,24 @@ party) while the TLS server is co-located with the TEE-hosted
 confidential workload (the attester).
 
 The flow starts with the client initiating a verification session with a
-trusted verifier.  The verifier returns the kinds of evidence it
+trusted verifier.  The verifier returns the evidence types it
 understands and a nonce that will be used to challenge the attester.
 
 The client starts the TLS handshake with the server by supplying the
 attestation-related parameters it has obtained from the verifier.  If
 the server supports one of the offered evidence types, it will echo it
-in the specular extension and proceed by invoking the local API to
-request the attestation.  The returned evidence binds the identity key
-with the platform identity and security state.  The server
-then signs the handshake transcript with the (attested) identity key,
-and sends the attestation evidence together with the signature over to
-the client.
+in the specular extension and proceed by invoking the local API
+(`attest_key(...)`) to request attestation using the nonce supplied by
```

Comment:
I assume the API name `attest_key` is not standardized. Should we use it here?

Reply:
FYI.  In a private exchange, Yuxuan Song raised the same point.

Reply:
Apologies, I thought I'd replied to this. I've added some text next to `attest_key` to make it clear that it's just a way for us to denote the attestation process in the diagram. Does that work?